JoinColony/colonyCDapp
# List actions query

## Description

This PR adds `GetColonyActions` query and wires it in with `ColonyActions` and related components.

<img width="727" alt="image" src="https://user-images.githubusercontent.com/112586815/216847053-fef55bdc-8262-4c9b-9add-a32efed9f4b4.png">

Pagination will be added in #245.

## Testing
The ingestor image has been modified to run [block-ingestor#11](https://github.com/JoinColony/block-ingestor/pull/11), so you can start everything as usual.
1. Create a colony (you can't use the auto created ones to mint tokens) **NOTE**: The extension adding step will fail, just ignore it and go to the created colony
2. Click on the Test Mint Tokens button a few times.
3. Verify the actions appear in the list after reloading. The sorting should work, but it'll be reworked in #245.

Resolves #205
